js/models/account.js

```diff
@@ -74,6 +75,17 @@ define(function(require) {
 			}
 			return undefined;
 		},
+		getSpecialFolder: function() {
+			if (!this.folders) {
+				return undefined;
+			}
+			return _.find(this.folders, function(folder) {
+				// TODO: handle special folders in subfolder properly
```

Comment:
Then maybe apply this globally, e.g. search for `// TODO` and replace it with `// @TODO`. And same with multi-line comments.